components/TextIamgeTransparent.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+    <div class="textimagetransparent">
+        <img class="image" :src="image" />
```

Comment:
`img` tags should be `NuxtImg` tags, like `<NuxtImg src="/images/gamers.jpg" />`